### ISSUE TITLE
[a11y]: ComboBox tooltips are not displayed for keyboard users

### ISSUE THREAD

**Issue by patrickobrien1** [carbon-design-system/carbon#20618]
### Package

@carbon/react

### Browser

Firefox

### Operating System

MacOS

### Package version

1.76.0

### React version

16.14.0

### Automated testing tool and ruleset

AVT Audit

### Description

When the ComboBox receives keyboard focus, it should display the tooltip visually so that users who use the keyboard are able to view the full string if it is too long for the ComboBox. Tool tip is not shown for any of the selected values in all editable fields after receiving Keyboard focus. ComboBox values are getting truncated due to the length, and tool tip is shown with mouse hover. Expecting the same behavior with keyboard focus.

### WCAG 2.1 Violation

 1.1.1 Non Text Content

### Reproduction/example

https://react.carbondesignsystem.com/?path=/story/components-combobox--default

### Steps to reproduce

1. Select the first option in the combo box 'An example option that is really long to show what should be done to handle long text'
2. Provide keyboard focus to the ComboBox.
3. Note lack of tooltip indication

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by nehaaprasad** [comment]
@patrickobrien1 
fixed it let me know

**Comment by lawrencepobrien** [comment]
I am no longer able to reproduce this issue in the consuming service or the storybook. Please close this issue as completed.
